src/crud.py

```
@@ -1551,3 +1551,56 @@ async def get_duplicate_documents(
+        session_id: Public ID of the session
+        
+    Returns:
+        DeriverStatus: Schema containing processing counts and status
+    """
+    # Get session internal ID from public_id
+    session = await get_session(db, app_id=app_id, session_id=session_id, user_id=user_id)
+    
+    # Count unprocessed messages
+    unprocessed_stmt = select(func.count()).select_from(models.QueueItem).where(
+        models.QueueItem.session_id == session.id,
+        models.QueueItem.processed == False
```

Ruff (0.11.9)

Avoid equality comparisons to `False`; use `if not models.QueueItem.processed:` for false checks

Replace with `not models.QueueItem.processed`

(E712)